### ISSUE TITLE
Multi-word icon names not working

### ISSUE THREAD

**Issue by max** [google/material-design-icons#177]
Perhaps I'm not doing this right but multi-word icons (e.g. `arrow drop down`) don't seem to render. Here is a JSBin example:

http://jsbin.com/boleno/edit?html,css,output

See how single word icons like `face` work but `arrow drop down` doesn't.

**Comment by max** [comment]
Oh, it looks like `arrow_drop_down` works. Is this explained in the docs somewhere? If not I'm happy to add a disclaimer.

**Comment by shyndman** [comment]
You're right. It doesn't look like the documentation mentions this, and it probably should.

If you take a look at a specific icon on [google.com/design/icons](https://www.google.com/design/icons/#ic_flight_takeoff), you'll notice the "icon font" button on the bottom drawer. When clicked, the ligature and codepoint for the selected icon will be displayed.

**Comment by shyndman** [comment]
Now tracking in #184 under a new name. Closing this issue.

**Comment by Aratillo** [comment]
> Oh, it looks like `arrow_drop_down` works. Is this explained in the docs somewhere? If not I'm happy to add a disclaimer.

I know it's been 7 years but that post there really helped me out thank you!

**Comment by Retroguru** [comment]
Issue still exists. Also commented it in #184
